articles/quickstart/native/react-native/_includes/_login.md

```diff
@@ -0,0 +1,99 @@
+The first step in adding authentication to your application is to provide a way for your users to log in. The fastest, most secure, and most feature-rich way to do this with Auth0 is to use the [login page](https://auth0.com/docs/hosted-pages/login).
```

Comment:
In every other QS it is `hosted`, The only QS that uses `centralized` is the Android one 😄 
However the filename is often `_centralized_login.md` in the includes.